packages/lesswrong/components/common/LWPopper.tsx

```diff
@@ -84,55 +98,77 @@ const LWPopper = ({
     }
   ];
 
+  // Custom modifier to check for a second tooltip
+  const secondTooltipModifier = checkForSecondTooltip ? [
```

Comment:
Is this meant to be here? It looks like maybe this was a first attempt before realising `fallbackPlacements` existed that should then have been removed since `checkForSecondTooltip` isn't set anywhere?

If it _is_ meant to be here, I think it probably won't work correctly since you're setting the exact same `id` on every tooltip (which is undefined behaviour) and this will probably just trigger for every tooltip where `checkForSecondTooltip` is set.